hebrewreader.py

```diff
@@ -300,7 +329,7 @@ def main():
         templates['postvoca'] = args.post_voca_tex.read()
         generate(args.passages,
                 args.include_voca, args.combine_voca, args.clearpage_before_voca,
-                args.large_text, False, args.tex, args.pdf, templates)
+                args.large_text, False, args.tex, args.pdf, args.lang, templates)
```

Comment:
Does this work? The `p_misc` parser did not get a `lang` option, so I would expect `args.lang` to always be `None`. I would expect to be able to do something like (cf the readme):

```sh
./hebrewreader.py --lang greek --pdf genesis.pdf Genesis
```